### PR TITLE
Issue #7759: Regexp & RegexpSinglelineJava check does not support SuppressionXpathFilter.

### PR DESCRIPTION
**Pull request by DXTkastb**
Fixes: #7759 & #7760 

Website view:
poi

@rnveach @romani Minor issue. please review

### CODE REVIEW

**Comment by romani** [comment]
@DXTkastb , please rebase to latest code just in case. I will start review soon.

File: src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java [hunk 1]
```diff
@@ -89,6 +89,9 @@
  * VariableDeclarationUsageDistance
  * </li>
  * </ul>
+ * <p>Reason for RegexpSinglelineJava and Regexp support unavailability is at
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java:92]: as I requested in https://github.com/checkstyle/checkstyle/issues/7759#issuecomment-605525287

please update lines:
```
 * Regexp
 * </li>
 * <li>
 * RegexpSinglelineJava
 * </li>
 * <li>
```

to:
```
 * Regexp (reason is at <a href="https://github.com/checkstyle/checkstyle/issues/7759#issuecomment-605525287"> #7759</a>)
 * </li>
 * <li>
 * RegexpSinglelineJava (reason is at <a href="https://github.com/checkstyle/checkstyle/issues/7759#issuecomment-605525287"> #7759</a>)
 * </li>
 * <li>
```

the same for `config_filters.xml`.

Comment (DXTkastb) [reply src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java]: done

Comment (DXTkastb) [reply src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java]: I have posted the website image in the first post

**Pull request merged**